projects/slap_manipulation/assets/june_2nd_experiment.yaml

```diff
@@ -0,0 +1,34 @@
+test:
+- 05-23_17-48-49_0
+- 05-23_17-47-26_0
+- 2023-05-30_16-16-07_index3
+- 2023-05-30_16-18-21_index6
+- 2023-05-30_16-24-35_index15
+train:
+- 05-23_17-39-18_0
+- 05-23_17-44-19_0
+- 05-23_17-35-59_0
+- 05-23_17-43-01_0
+- 05-23_17-31-20_0
+- 05-23_17-53-58_0
+- 05-23_17-45-45_0
+- 05-23_17-51-17_0
+- 05-23_17-37-36_0
+- 05-23_17-49-59_0
+- 2023-05-30_16-19-37_index8
+- 2023-05-30_16-20-53_index10
+- 2023-05-30_16-21-36_index11
```

Comment:
i'm not sure how much i like actually committing these files to the repo - will others be able to use them? are we releasing our data?

Reply:
Fair point. I will remove these, let's figure out data requirements and we can release with that.